cmake/configs/default.cmake

```diff
@@ -178,7 +178,7 @@ hunter_config(geos VERSION 3.4.2)
 hunter_config(gflags VERSION 2.2.1)
 hunter_config(glbinding VERSION 2.1.3-p0)
 hunter_config(glew VERSION 2.0.0)
-hunter_config(glfw VERSION 3.3.0-p4)
+hunter_config(glfw VERSION 3.2.1)
```

Comment:
I don't expect downgrading, please remove.

Reply:
@ruslo it seems that "3.2.1" is latest release: https://github.com/glfw/glfw/releases

Or "3.3.0-p4" is a local release of hunter's fork? What shall I do?

Reply:
> Or "3.3.0-p4" is a local release of hunter's fork?

Here is pull request with 3.3.0 version: https://github.com/hunter-packages/glfw/pull/1

@headupinclouds I can't see 3.3.0 tag. I guess 3.3.0 is next release from master, right?

Reply:
So... I must somehow load glfw's release and rename its tag and release number?

Reply:
> I must somehow load glfw's release and rename its tag and release number?

Not sure what you mean.

Also why do you need `3.2.1` release in the first place?
Official `3.2.1` release date is 18 Aug 2016: https://github.com/glfw/glfw/releases/tag/3.2.1
Fork `3.3.0` release date is 22 Aug **2017**: https://github.com/hunter-packages/glfw/releases/tag/3.3.0-p4

I expect all changed you need is already in `3.3.0` release.

Reply:
@ruslo they add gamepads and  SDL game controller database
https://github.com/glfw/glfw/issues/1117

Reply:
What commit?

> https://github.com/glfw/glfw/blob/2867ca1e5b003b167fac33e8136a4698bb3aa074/include/GLFW/glfw3.h

Commit `2867ca1e5b003b167fac33e8136a4698bb3aa074` is in branch `master` and not released https://github.com/glfw/glfw/commit/2867ca1e5b003b167fac33e8136a4698bb3aa074

Reply:
@ruslo LOL I'm feeling stupid now :DDD